dataland-frontend/src/App.vue

```diff
@@ -65,12 +65,3 @@ export default defineComponent({
   },
 });
 </script>
-
-<style lang="scss">
-@import "./assets/css/main.css";
-@import "./assets/css/variables";
```

Comment:
The "active" colors differ:
![image](https://user-images.githubusercontent.com/47553592/210359082-2e279f9b-d6fe-43e1-aca3-a2dbcfd27bf6.png)

Reply:
understood. ok like that.